.circleci/config.yml

```diff
@@ -297,6 +300,8 @@ jobs:
           name: Run e2e tests (Puppeteer and Jest)
           working_directory: ~/workbench/e2e
           command: yarn test:ci $(circleci tests glob "tests/**/*.spec.ts" | circleci tests split --split-by=timings --timings-type=classname --show-counts)
+      - store_test_results:
+          path: ~/workbench/api/build/test-results/integrationTest/
```

Comment:
Are you sure this works for the e2e test results? The e2e (JUnit) test results are explicitly set to write to `/e2e/logs` dir. And, `store_test_results` key is called again at line #308. I don't think circleci allows duplicate `store_test_results` keys in same job.

Reply:
whoops. good catch.

Is the TESTS UI tab working for this workflow?